### ISSUE TITLE
Crash after renaming theme file

### ISSUE THREAD

**Issue by Lineflyer** [cgeo/cgeo#10093]
While testing theme sync yesterday I had several crashes with 2021.02.28-NB.

Sadly I cannot really reproduce it up to now.
What I potentially did before this happened was:
- Debug mode active
- End c:geo
- Rename some theme folder and/or theme file
- Start c:geo
- Open live map
- Open theme selection menu

Not sure whether it crashed on startup, on live map or only when opening the theme selection.

This is the log I exported yesterday:
```
02-28 23:18:51.730 17335 17335 E AndroidRuntime: FATAL EXCEPTION: main
02-28 23:18:51.730 17335 17335 E AndroidRuntime: Process: cgeo.geocaching, PID: 17335
02-28 23:18:51.730 17335 17335 E AndroidRuntime: java.lang.NullPointerException: Attempt to read from field 'android.net.Uri cgeo.geocaching.storage.ContentStorage$FileInformation.uri' on a null object reference
02-28 23:18:51.730 17335 17335 E AndroidRuntime: 	at cgeo.geocaching.maps.mapsforge.v6.RenderThemeHelper.getThemeFor(RenderThemeHelper.java:183)
02-28 23:18:51.730 17335 17335 E AndroidRuntime: 	at cgeo.geocaching.maps.mapsforge.v6.RenderThemeHelper.reapplyMapTheme(RenderThemeHelper.java:134)
02-28 23:18:51.730 17335 17335 E AndroidRuntime: 	at cgeo.geocaching.maps.mapsforge.v6.NewMap.switchTileLayer(NewMap.java:642)
02-28 23:18:51.730 17335 17335 E AndroidRuntime: 	at cgeo.geocaching.maps.mapsforge.v6.NewMap.initializeLayers(NewMap.java:714)
02-28 23:18:51.730 17335 17335 E AndroidRuntime: 	at cgeo.geocaching.maps.mapsforge.v6.NewMap.onStart(NewMap.java:709)
02-28 23:18:51.730 17335 17335 E AndroidRuntime: 	at android.app.Instrumentation.callActivityOnStart(Instrumentation.java:1435)
02-28 23:18:51.730 17335 17335 E AndroidRuntime: 	at android.app.Activity.performStart(Activity.java:8222)
02-28 23:18:51.730 17335 17335 E AndroidRuntime: 	at android.app.ActivityThread.handleStartActivity(ActivityThread.java:3818)
02-28 23:18:51.730 17335 17335 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.performLifecycleSequence(TransactionExecutor.java:221)
02-28 23:18:51.730 17335 17335 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.cycleToPath(TransactionExecutor.java:201)
02-28 23:18:51.730 17335 17335 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:173)
02-28 23:18:51.730 17335 17335 E AndroidRuntime: 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:97)
02-28 23:18:51.730 17335 17335 E AndroidRuntime: 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2307)
02-28 23:18:51.730 17335 17335 E AndroidRuntime: 	at android.os.Handler.dispatchMessage(Handler.java:106)
02-28 23:18:51.730 17335 17335 E AndroidRuntime: 	at android.os.Looper.loop(Looper.java:246)
02-28 23:18:51.730 17335 17335 E AndroidRuntime: 	at android.app.ActivityThread.main(ActivityThread.java:8506)
02-28 23:18:51.730 17335 17335 E AndroidRuntime: 	at java.lang.reflect.Method.invoke(Native Method)
02-28 23:18:51.730 17335 17335 E AndroidRuntime: 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:602)
02-28 23:18:51.730 17335 17335 E AndroidRuntime: 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1130)
02-28 23:18:51.730 17335 17335 W cgeo    : [main] UncaughtException
02-28 23:18:51.730 17335 17335 W cgeo    : java.lang.NullPointerException: Attempt to read from field 'android.net.Uri cgeo.geocaching.storage.ContentStorage$FileInformation.uri' on a null object reference
02-28 23:18:51.730 17335 17335 W cgeo    : 	at cgeo.geocaching.maps.mapsforge.v6.RenderThemeHelper.getThemeFor(RenderThemeHelper.java:183)
02-28 23:18:51.730 17335 17335 W cgeo    : 	at cgeo.geocaching.maps.mapsforge.v6.RenderThemeHelper.reapplyMapTheme(RenderThemeHelper.java:134)
02-28 23:18:51.730 17335 17335 W cgeo    : 	at cgeo.geocaching.maps.mapsforge.v6.NewMap.switchTileLayer(NewMap.java:642)
02-28 23:18:51.730 17335 17335 W cgeo    : 	at cgeo.geocaching.maps.mapsforge.v6.NewMap.initializeLayers(NewMap.java:714)
02-28 23:18:51.730 17335 17335 W cgeo    : 	at cgeo.geocaching.maps.mapsforge.v6.NewMap.onStart(NewMap.java:709)
02-28 23:18:51.730 17335 17335 W cgeo    : 	at android.app.Instrumentation.callActivityOnStart(Instrumentation.java:1435)
02-28 23:18:51.730 17335 17335 W cgeo    : 	at android.app.Activity.performStart(Activity.java:8222)
02-28 23:18:51.730 17335 17335 W cgeo    : 	at android.app.ActivityThread.handleStartActivity(ActivityThread.java:3818)
02-28 23:18:51.730 17335 17335 W cgeo    : 	at android.app.servertransaction.TransactionExecutor.performLifecycleSequence(TransactionExecutor.java:221)
02-28 23:18:51.730 17335 17335 W cgeo    : 	at android.app.servertransaction.TransactionExecutor.cycleToPath(TransactionExecutor.java:201)
02-28 23:18:51.730 17335 17335 W cgeo    : 	at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:173)
02-28 23:18:51.730 17335 17335 W cgeo    : 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:97)
02-28 23:18:51.730 17335 17335 W cgeo    : 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2307)
02-28 23:18:51.730 17335 17335 W cgeo    : 	at android.os.Handler.dispatchMessage(Handler.java:106)
02-28 23:18:51.730 17335 17335 W cgeo    : 	at android.os.Looper.loop(Looper.java:246)
02-28 23:18:51.730 17335 17335 W cgeo    : 	at android.app.ActivityThread.main(ActivityThread.java:8506)
02-28 23:18:51.730 17335 17335 W cgeo    : 	at java.lang.reflect.Method.invoke(Native Method)
02-28 23:18:51.730 17335 17335 W cgeo    : 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:602)
02-28 23:18:51.730 17335 17335 W cgeo    : 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1130)
```

**Comment by eddiemuc** [comment]
Stacktrace should be sufficient to track and fix this. Might be a race condition where file was accessed while it was also changed/deleted by async background sync task, combined with insufficient NPE guarding.

I will fix this

**Comment by eddiemuc** [comment]
Should be fixed with PR #10094
